db.js

```diff
@@ -1,5 +1,5 @@
 const { Client } = require('pg');
-const settings = require('./settings.json');
+const settings = require('./settings.test.json');
```

Comment:
Really trying to not change this in code -- for example, right now, if we merged this PR, it would force EVERYONE to change the name of their settings file. Just keep multiple files and overwrite `settings.json` with the one you want to use